packages/wonder-blocks-tooltip/index.test.js

```diff
@@ -9,7 +9,12 @@ describe("@khanacademy/wonder-blocks-tooltip", () => {
 
         // Assert
         expect(Object.keys(result).sort()).toEqual(
-            ["default", "TooltipContent"].sort(),
+            [
+                "default",
+                "TooltipContent",
+                "TooltipPopper",
+                "TooltipTail",
+            ].sort(),
```

Comment:
I think we can replace this with `[
                "TooltipContent",
                "TooltipPopper",
                "TooltipTail",
                "default",
            ]` and get rid of the call to `sort()` here

Reply:
This is a pattern we are using on all packages to avoid injecting manual ordering issues, so I'd recommend to leave it for consistency. What do you think?

Reply:
I see, If it’s to ensure consistency I think it’s fine to keep

Reply:
Yeah, this was here to help us maintain control over what we export without being overly picky about the ordering so that we dodge busywork if the order changed.